src/test/java/no/unit/nva/model/PublicationTest.java

```diff
@@ -72,6 +74,14 @@ public void updatingDoiRequestStatusWithoutDoiRequestThrowsIllegalStateException
             is(equalTo(Publication.ERROR_MESSAGE_UPDATEDOIREQUEST_MISSING_DOIREQUEST)));
     }
 
+    @Test
+    public void getModelVersionReturnsModelVersionDefinedByGradle()
```

Comment:
Tests can be/is package private in Junit5

Reply:
This class has some protected methods and package private tests will mess the order of the functions (tests first, helper methods afterwards)